rinja_parser/src/lib.rs

```diff
@@ -425,15 +459,35 @@ fn str_lit(i: &str) -> ParseResult<'_> {
     Ok((i, s.unwrap_or_default()))
 }
 
+fn str_lit(i: &str) -> Result<(&str, StrLit<'_>), ParseErr<'_>> {
+    let (i, (prefix, content)) =
+        tuple((opt(alt((char('b'), char('c')))), str_lit_without_prefix))(i)?;
+    let prefix = match prefix {
+        Some('b') => StrPrefix::Binary,
+        Some('c') => StrPrefix::CLike,
+        _ => StrPrefix::None,
+    };
+    Ok((i, StrLit { prefix, content }))
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct CharLit<'a> {
+    pub has_b_prefix: bool,
```

Comment:
Could you please make this a `prefix: CharPrefix` field analogous to `StrLit`? I am hopeful that sometime soon ASCII literals like `a"Hello"` or `a'!'` are introduced.

Reply:
Coherency and also easier for future updates. Love it!